shared/js/devtools/panel.js

```diff
@@ -125,6 +125,18 @@ const actionHandlers = {
             table.appendChild(row)
         }
     },
+    error: (m) => {
+        const { documentUrl, message, filename, lineno, colno, stack, scriptOrigins } = m.message
+        const row = document.getElementById('cookie-row').content.firstElementChild.cloneNode(true)
+        const cells = row.querySelectorAll('td')
+        cells[1].textContent = documentUrl
+        cells[2].textContent = `JS🪲 ${message}`
+        cells[3].textContent = scriptOrigins.join(',')
+        if (stack) appendCallStack(cells[3], stack, 0)
+        cells[4].textContent = `${filename}:${lineno}:${colno}`
```

Comment:
Is it intentional that this is in the `type` field? It's probably fine there in terms of position, the column name seems to be getting misaligned from content though.